lib/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -68,26 +74,39 @@ export const DateTimePickerToolbar: React.FC<ToolbarComponentProps> = ({
           <Grid
             item
             container
-            xs={6}
+            xs={hasSeconds ? 9 : 6}
             justify="center"
             alignItems="flex-end"
             direction={rtl ? 'row-reverse' : 'row'}
           >
             <ToolbarButton
-              variant="h3"
+              variant={timeTypographyText}
```

Comment:
This could be an opportunity to fix the `<hx>` DOM output

Reply:
What do you mean by this?